admin/src/components/signup-search.tsx

```diff
@@ -0,0 +1,438 @@
+import {
```

Comment:
Don’t copy paste an entire component, of you need to reuse something refactor it appropriately.

Reply:
This is a fairly significant refactor of `signup-list.tsx`, for example search terms are stored in component state, not in routes. And a `Select` component with `tags` enabled is preferred over parsing search input.

Is there something specific you're looking at?

I would agree that, the list of signups could conceivably be made its own component and shared between these two pages, along with the approval / reject options.

Reply:
The approve/reject logic and table rendering to start, the majority of this code should be refactored so it can be shared between the components.

But I don't get why this additional page is added, I see no new functionality added by this besides separating the `filter:` into *six* different input fields. Is there a user story describing this somewhere?

Btw the point of keeping the search terms in the routes is so that you can link to a search, e.g. https://signup.steemit.com/admin/signups?q=status%3Amanual_review+hello